ci-operator/step-registry/aws/deprovision/cco-manual-users/static/aws-deprovision-cco-manual-users-static-ref.yaml

```diff
@@ -1,9 +1,6 @@
 ref:
   as: aws-deprovision-cco-manual-users-static
-  from_image:
-    namespace: ocp
-    name: "4.5"
```

Comment:
don't you need to pin a recent 4.y if you want to rely on `openshift/installer#6008`?  Otherwise 4.8 flows using this step, etc., will fail to find the binary, right?

Reply:
To be honest, I am a bit confuse with the difference between:
```
from: upi-installer
```
v.s
```
  from_image:
    namespace: ocp
    name: "4.5"
    tag: upi-installer
```
My understanding is `from: upi-installer` will rely on the prow ci jobs where defined `upi-installer`, right? If yes, when some jobs did not defined `upi-installer`, the job referencing the step will be broken as what you said. If that, I probably need to pin it to `4.12`.

Reply:
For the specific step, presently it is only consumed by QE's jobs, the job we defined `upi-installer` like https://github.com/openshift/release/blob/master/ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.12.yaml#L14-L17, that is supposed to works in 4.12 prow jobs. Of course, if we want to make the steps working in broader jobs, maybe it is most safe to pin it to `4.12` tag. WDYT?

Reply:
I changed back to use a pinned-version (4.12) upi-installer image for the step.